src/runtime/runtime_api.cpp

```diff
@@ -148,16 +147,16 @@ extern "C" __attribute__((used)) void *halide_runtime_api_functions[] = {
     (void *)&halide_opencl_set_device_type,
     (void *)&halide_opencl_set_platform_name,
     (void *)&halide_opencl_wrap_cl_mem,
-    (void *)&halide_opengl_context_lost,
-    (void *)&halide_opengl_create_context,
-    (void *)&halide_opengl_detach_texture,
-    (void *)&halide_opengl_device_interface,
-    (void *)&halide_opengl_get_proc_address,
-    (void *)&halide_opengl_get_texture,
-    (void *)&halide_opengl_initialize_kernels,
-    (void *)&halide_opengl_run,
-    (void *)&halide_opengl_wrap_render_target,
-    (void *)&halide_opengl_wrap_texture,
+    // (void *)&halide_opengl_context_lost,
```

Comment:
Why comment and not delete?

Reply:
...scalpel left in patient, sorry, fixing

Reply:
Actually, halide_opengl_create_context and halide_opengl_get_proc_address still appear to be needed by OpenGLCompute